stdlib/public/Concurrency/Task.swift

```diff
@@ -430,36 +435,26 @@ public func detach<T>(
 
 /// Run given throwing `operation` as part of a new top-level task.
 ///
-/// Creating detached tasks should, generally, be avoided in favor of using
-/// `async` functions, `async let` declarations and `await` expressions - as
-/// those benefit from structured, bounded concurrency which is easier to reason
-/// about, as well as automatically inheriting the parent tasks priority,
-/// task-local storage, deadlines, as well as being cancelled automatically
-/// when their parent task is cancelled. Detached tasks do not get any of those
-/// benefits, and thus should only be used when an operation is impossible to
-/// be modelled with child tasks.
-///
-/// ### Cancellation
-/// A detached task always runs to completion unless it is explicitly cancelled.
-/// Specifically, dropping a detached tasks `Task.Handle` does _not_ automatically
-/// cancel given task.
+/// If the operation throws an error, this method rethrows that error.
```

Comment:
This is quite not true, `detach` is not throwing.

What happens is that if the operation throws, then the returned Task.Handle's `try await get()` will throw. This is "kind of" like rethrowing but I don't think we can use that word here since "rethrows" is a specific thing in Swift.

Reply:
I'm changing "rethrows" to "propagates", to match what we call this in TSPL where the error makes its way back through a series of calls to throwing functions.

https://docs.swift.org/swift-book/LanguageGuide/ErrorHandling.html#ID510